src/index.css

```diff
@@ -75,6 +75,25 @@ body {
   padding-left: 35px;
 }
 
+.highlight-search-term {
+  color: #fff;
+  background-color: #000;
+}
+
+.selected-contacts-list {
+  padding: 0;
+}
+
+.selected-contacts-list p {
+  text-align: center;
+}
+
+.selected-contact {
+  background-color: #B9F6CA;
+  border-color: #fff;
+  padding: 2% 2%;
+}
```

Comment:
CSS should ideally be placed in the corresponding css file of the react component.